### DISCUSSION TITLE
use more than one column in playback view

### DISCUSSION THREAD

**Discussion by L0ric0** [jcorporation/myMPD#482] [Ideas]
In the playback view, all options that are shown are added under the previous ones. On wide screens that does not look good, as only halve the page is used and to see the things at the botom one has to scrool.

It would be nice if multiple coulumns could be created dynamicaly.

**Comment by jcorporation** [comment]
Sorry for the very late response. Commit [f2ce396](https://github.com/jcorporation/myMPD/commit/f2ce396a8dc8ef1a50fd5e925bed4d07c9dc94a5) adds a second column for large screens.
